pkg/store/zookeeper.go

```diff
@@ -131,9 +131,13 @@ func (s *Zookeeper) Watch(key string, stopCh <-chan struct{}) (<-chan *KVPair, e
 	go func() {
 		defer close(watchCh)
 
-		// GetW returns the current value before setting the watch.
-		watchCh <- &KVPair{key, resp, uint64(meta.Version)}
+		// Get returns the current value before setting the watch.
```

Comment:
It is, we still send back the first value to the channel but this time using a simple `Get` ;)